src/System.Numerics.Tensors/src/Configurations.props

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PackageConfigurations>
+      netstandard;
```

Comment:
Why do we need a `netstandard` (no version) configuration?

Reply:
I believe @ericstj explained it as being needed for anything in CoreFX that depends on this assembly.

Reply:
We shouldn't have anything else in CoreFX depending on this assembly.

Reply:
Its for source/vertical build to work.  We could decide to leave it out of the package.  We need to build it because we only run tests during the vertical builds.